repository/scripts/widget/frame/world-map.js

```diff
@@ -43,11 +44,11 @@ module.exports = (function () {
 			widget.setEvents(ctx.player, 1422, 45, 2, 2, 2);
 			widget.setEvents(ctx.player, 1422, 46, 2, 2, 2);
 			widget.setEvents(ctx.player, 1422, 47, 2, 2, 2);
-			//varc(ctx.player, 622, getCoordHash(map.getCoords(ctx.player)));
+			varc(ctx.player, 622, ENGINE.getCoordHash(map.getCoords(ctx.player)));
```

Comment:
You shouldn't need the `ENGINE.getCoordHash()` part here (just `map.getCoords(ctx.player)` should be enough). The `varc()` method should automatically convert it for you (if it doesn't, let me know & I'll add an engine method)